### ISSUE TITLE
Uncaught (in promise) IntegrationError: Invalid value for stripe.confirmCardPayment intent secret: value should be a client_secret string. You specified: true.

### ISSUE THREAD

**Issue by TiagoPerdigao1983** [CleverProgrammers/react-challenge-amazon-clone#2]
Hi guys. Your app doesn't work on a fresh install. This error occurs: "Unhandled Rejection (IntegrationError): Invalid value for stripe.confirmCardPayment intent secret: value should be a client_secret string. You specified: true." Any ideas on how to solve it.

**Comment by gfcf14** [comment]
@TiagoPerdigao1983 were you able to solve this? I have the same issue

**Comment by shanuv000** [comment]
I have the same issue please help.

**Comment by chadrackkyungu** [comment]
@shanuv000 @gfcf14 @TiagoPerdigao1983  

ooooooooooooooooooohhhhhhhhhhh i find out why, here is the process

Go to your functions folder and run : firebase emulators:start
make sure you don't have it open in another window, after it opened run the frontend which is amazon-clone then then try to buy and process your orders, it will work perfect.
i spent 2 days trying to figure out but thanks God i got it now i can move on, hope it will help some one, if you have any question don't hesitate asked me i will help you.

**Comment by johntranz** [comment]
I tried your solution step above, but still can't fix this error

**Comment by DKD-tech** [comment]
Hi, for my case I went to this site: https://stripe.com/docs/stripe-js/react.  there is a brief explanation of how to use the card element. And I changed the name of my const:  promise in app.js to stripePromise  and it worked.

Salut , pour mon cas je suis allé sur ce site: https://stripe.com/docs/stripe-js/react , et il y'a une bref explication de comment utiliser react js stripe , et j'ai changé ma const: promise dans le fichier App.js par stripePromise et ça à fonctionné.  Je pense que le serveur avait enregistré la trace de promise c'est pour cela .
